Userland/reboot.cpp

```diff
@@ -0,0 +1,11 @@
+#include <LibC/unistd.h>
+
+
+int main(int, char**)
+{
+    reboot();
```

Comment:
We should print an error message if `reboot()` fails (e.g when run as non-superuser.)

Reply:
I've given it the classic non-root error message ;)